clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -89,6 +89,10 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
         description.labels.put("interestingHealthProviderNames", description.interestingHealthProviderNames.join(",") )
       }
 
+      if (description.addSecureSignature == false) {
```

Comment:
does this need a null check? I see there is a default value on the field but it is a big b boolean so..